a1/src/main/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/stackoverflow
```

Comment:
You already have the schema in the JDBC url. As such, in the Flyway migrations, you should not need to:
 - Create the schema,
 - Explicitly name the schema when creating tables.

Usually it is good to not have schema names in your Flyway scripts, because then you have greater flexibility and can move your application easier between environments.